crawl-ref/source/art-data.txt

```diff
@@ -647,13 +648,15 @@ TILE:     urand_cutlass
 TILE_EQ:  capt_cutlass
 BRAND:    SPWPN_SPEED
 
+ENUM:     STORM_BOW
```

Comment:
This line is superfluous, enum names are automatically generated from `NAME` unless there's a need not to. It's better to not add one.